views/partials/artist.hbs

```diff
@@ -0,0 +1,3 @@
+<a href="/albums/{{id}}}">View Albums</a>
+<p>{{name}}</p>
+<img src="{{images.url}}" alt="">
```

Comment:
Remember that you might get multiple images with your request, you should pinpoint the first image of the array.